systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -112,6 +117,44 @@ void testKafkaConnectIoNetwork() {
         assertThat("Kafka Connect IO network count doesn't match expected value", values.stream().mapToDouble(i -> i).sum() > 0);
     }
 
+    @Test
+    void testKafkaExporterDataAfterExchange() throws Exception {
+        createTestMethodResources();
+        testMethodResources().deployKafkaClients(CLUSTER_NAME + "-" + Constants.KAFKA_CLIENTS).done();
+
+        availabilityTest(5000, CLUSTER_NAME, TEST_TOPIC_NAME);
+
+        kafkaExporterMetricsData = collectKafkaExporterPodsMetrics();
+        assertThat("Kafka Exporter metrics are empty", kafkaExporterMetricsData.size() > 0);
```

Comment:
`assertThat("Kafka Exporter metrics are empty"` is the wrong way around. You're asserting that they're not empty. We can reword the message to avoid ambiguity about what is actually true and what we expect to be true like this:

```suggestion
        assertThat("Kafka Exporter metrics should be non-empty", kafkaExporterMetricsData.size() > 0);
```